src/index.js

```diff
@@ -1,10 +1,42 @@
-function component() {
-  const element = document.createElement('div');
+import './modules/style.css';
 
-  // Lodash, currently included via a script, is required for this line to work
-  // element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+function showToDo(todo) {
+  const list = document.querySelector('.list');
+  const newlist = document.createElement('li');
+  newlist.classList.add('newlist');
+  newlist.innerHTML = `
+    <input id="${todo.index}" class="check" type="checkbox"/>
+    <span class="text">${todo.description}</span>
+    <button class="delete"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M10.001 7.8a2.2 2.2 0 1 0 0 4.402A2.2 2.2 0 0 0 10 7.8zm0-2.6A2.2 2.2 0 1 0 9.999.8a2.2 2.2 0 0 0 .002 4.4zm0 9.6a2.2 2.2 0 1 0 0 4.402 2.2 2.2 0 0 0 0-4.402z"/></svg></button>
+    `;
+  list.append(newlist);
+}
 
-  return element;
+const todoList = [];
```

Comment:
Kindly include some tasks in the array in order to test the dynamic rendering where you could mark some tasks as completed and others uncompleted. This will help to test out your functionalities.

![image](https://user-images.githubusercontent.com/20554583/193407638-e6e939ce-5468-425b-a4a1-fb6b0399c0ab.png)

Reply:
@FaranoshAmini Here's an example of initializing some demo tasks:


> const todoList = [
>   {
>     description: 'Wash the dishes',
>     completed: false,
>     index: 1,
>   },
>   {
>     description: 'Eat unripe plantain',
>     completed: true,
>     index: 3,
>   },
>   {
>     description: 'Complete te todo list project',
>     completed: false,
>     index: 2,
>   },
>   {
>     description: 'Post a blog',
>     completed: true,
>     index: 3,
>   },
> 
> ];

Reply:
I included.